src/main/java/org/mifos/connector/fineractstub/model/SavingsDeposit.java

```diff
@@ -0,0 +1,179 @@
+package org.mifos.connector.fineractstub.model;
+
```

Comment:
Can you mark generated files with an annotation and exclude them from check style checks

Reply:
done